pkg/virt-operator/kubevirt_test.go

```diff
@@ -2147,20 +2147,20 @@ var _ = Describe("KubeVirt Operator", func() {
 
 			Expect(kvTestData.totalAdds).To(Equal(resourceCount - expectedUncreatedResources + expectedTemporaryResources))
 
-			Expect(len(kvTestData.controller.stores.ServiceAccountCache.List())).To(Equal(3))
-			Expect(len(kvTestData.controller.stores.ClusterRoleCache.List())).To(Equal(7))
-			Expect(len(kvTestData.controller.stores.ClusterRoleBindingCache.List())).To(Equal(5))
-			Expect(len(kvTestData.controller.stores.RoleCache.List())).To(Equal(3))
-			Expect(len(kvTestData.controller.stores.RoleBindingCache.List())).To(Equal(3))
-			Expect(len(kvTestData.controller.stores.CrdCache.List())).To(Equal(12))
-			Expect(len(kvTestData.controller.stores.ServiceCache.List())).To(Equal(3))
-			Expect(len(kvTestData.controller.stores.DeploymentCache.List())).To(Equal(1))
-			Expect(len(kvTestData.controller.stores.DaemonSetCache.List())).To(Equal(0))
-			Expect(len(kvTestData.controller.stores.ValidationWebhookCache.List())).To(Equal(3))
-			Expect(len(kvTestData.controller.stores.PodDisruptionBudgetCache.List())).To(Equal(1))
-			Expect(len(kvTestData.controller.stores.SCCCache.List())).To(Equal(3))
-			Expect(len(kvTestData.controller.stores.ServiceMonitorCache.List())).To(Equal(1))
-			Expect(len(kvTestData.controller.stores.PrometheusRuleCache.List())).To(Equal(1))
+			Expect(kvTestData.controller.stores.ServiceAccountCache.List()).To(HaveLen(3))
+			Expect(kvTestData.controller.stores.ClusterRoleCache.List()).To(HaveLen(7))
+			Expect(kvTestData.controller.stores.ClusterRoleBindingCache.List()).To(HaveLen(5))
+			Expect(kvTestData.controller.stores.RoleCache.List()).To(HaveLen(3))
+			Expect(kvTestData.controller.stores.RoleBindingCache.List()).To(HaveLen(3))
+			Expect(kvTestData.controller.stores.CrdCache.List()).To(HaveLen(12))
+			Expect(kvTestData.controller.stores.ServiceCache.List()).To(HaveLen(3))
+			Expect(kvTestData.controller.stores.DeploymentCache.List()).To(HaveLen(1))
+			Expect(kvTestData.controller.stores.DaemonSetCache.List()).To(HaveLen(0))
```

Comment:
Can `BeEmpty()` be applied here?

Reply:
oops. Done.

Reply:
@nunnatsa Hi. I still see `HaveLen(0)`, have you pushed the changes?

Reply:
@nunnatsa sorry for the noise, I didn't notice it is being changed in other commit

Reply:
Thank you @nunnatsa